src/test/main/java/com/spring/assignment/employeesystem/dao/EmployeeRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.spring.assignment.employeesystem.dao;
+
+import com.spring.assignment.employeesystem.entity.Employee;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface EmployeeRepository extends JpaRepository<Employee,Integer> {
+    List<Employee> findByUsername(String username);
+    Employee findById(int id);
```

Comment:
Repositories should either return lists or optional
What happens if you are searching for Employee with an id that does not exist in the db?